lib/stdlib/src/lists.erl

```diff
@@ -2113,7 +2113,12 @@ flatmap(F, List) when is_function(F, 1) ->
     flatmap_1(F, List).
 
 flatmap_1(F, [Hd | Tail]) ->
-    F(Hd) ++ flatmap_1(F, Tail);
+    case F(Hd) of
+        % The two first clauses are an optimization
```

Comment:
In order for the Erlang mode in Emacs to do proper indentation, comments at the beginning of a line should always be written as `%%`:

```suggestion
        %% The two first clauses are an optimization.
```

Reply:
Amended and force-pushed[29bb2b4](https://github.com/erlang/otp/pull/8749/commits/29bb2b45b3f067b7551dccbcbd6ab7f61bb93bae)

Reply:
Thanks! Added to our daily builds.